### DISCUSSION TITLE
🚀 Restarting the Community for BaGet Under a New Name and Repository

### DISCUSSION THREAD

**Discussion by seriouz** [loic-sharma/BaGet#780] [General]
In recent discussions on the BaGet repository, there's been talk about bringing the project back to life by restarting the community under a new name and repository. This idea comes up because it seems like the @loic-sharma BaGet repository has been stuck without updates / releases since [Sep 23, 2021](https://github.com/loic-sharma/BaGet/releases/tag/v0.4.0-preview2). Even though the author is active every day, he is not paying attention to the project or messages from the community.

One idea is to give the project a fresh start by creating a new name and repository. This might sound a bit surprising, but it makes sense. The main goal is to prevent any confusion with the existing BaGet project, especially on platforms like NuGet or Docker.

The whole point of a new name is to show that the community is determined to make things better and more active. The current project has been pretty quiet, and a new name could help show that things are changing. Plus, it's a chance to start over and work on making the project grow without any old problems holding it back.

What do you think about this idea? It's clear that the community's energy is still strong, and the suggestion to give the project a fresh start with a new name and repository could be a great way to keep the project moving forward.

🌟 Edit:
After the discussion we created a new fork of BaGet named **BaGetter** under this url: https://github.com/bagetter/BaGetter 🚀

**Comment by OndrejValenta** [comment]
I would go for it, we are discussing this for weeks. It's time for some action. 

What kind of name you have on your mind?

**Reply by seriouz** [reply]
I hate naming things. What about _NuGetter_?

**Reply by Regenhardt** [reply]
If it comes to this, I propose to just extend from the old name to _BaGetter_ to better signal the relationship.

**Comment by loic-sharma** [comment]
Hello, feel free to fork BaGet! I would be happy to answer questions on BaGet's implementation, NuGet and its protocol, or anything else. Feel free to reach out to me on [Discord](https://discord.gg/MWbhpf66mk) or on [Twitter](https://twitter.com/LoicSharma). Please do use a new name to distinguish your fork from BaGet.

For background, I left the NuGet team last year after burning out. I needed distance to recover and maintaining a NuGet side-project hindered that. I do plan to work on BaGet again in the future, but I will likely not add large new features like private feeds or V2 support. I would be open to give contributor access to folks with a proven track record.

Thank you for starting this conversation. I appreciate the energy of the community :)

> Plus, it's a chance to start over and work on making the project grow without any old problems holding it back.

Could you expand on what old problems you're referring to?

**Reply by seriouz** [reply]
Very nice to hear from you again. And that you don't want to give up the project.
With the problems, I generally meant the issues that are still open and suggestions that have been open for a long time.
Would something speak against it, if some people here from the community get admin rights, so that a clone under another name is not necessary?

**Reply by joestr** [reply]
I would also appreciate to give certain contributors more rights since the name and the project location are well-known.

**Reply by CPlusPlus17** [reply]
I think keeping the repo and the name should be archived when ever possibile. So I would appreciate also to add two or three contributors with rights to the repo.

**Reply by Regenhardt** [reply]
I definitely agree with enabling additional contributors to continue maintaining the current repository, as keeping the name and existing links is very helpful so everyone doesn't have to migrate.

**Reply by joestr** [reply]
@loic-sharma do you have someone in mind who fits the requirements?

**Reply by daghsentinel** [reply]
Forking the project and beginning anew under a different name could be the most appropriate course of action. Given the approximately 200 registered issues, starting afresh would involve re-registering only the most critical and essential ones. Additionally, the new owner(s) will have the freedom to configure the project according to their preferences. Importantly, the necessary approval has already been granted, and with no further responses from the current owner, it would be more productive to invest time in advancing the project rather than waiting for a reply.

**Reply by seriouz** [reply]
@daghsentinel Hi recently a new project was setup look here: https://github.com/bagetter/BaGetter
